core/src/main/web/app/mainapp/components/notebook/outputdisplay/outputdisplayfactory-service.js

```diff
@@ -82,12 +82,27 @@
           });
         },
         link: function(scope, element, attrs) {
+          var tagstofilter = ['applet', 'base', 'basefont', 'body', 'frame', 'frameset', 'head', 'html',
```

Comment:
Is there no lib to do this that we can use? This seems like something we do not want to roll on our own.